cfme/tests/configure/test_start_page.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import fauxfactory
+
+from cfme.base.credential import Credential
+from cfme.configure.settings import Visual
+from cfme.configure.access_control import Group, User
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+faux_value = fauxfactory.gen_alphanumeric()
```

Comment:
**REQUIRED:** There is no need for this to be at the modular level, your group is module scoped so that call to the function can just be inside there.